src/discurses/config.py

```diff
@@ -35,23 +53,27 @@ def create_dir(directory):
     avatar = await discord.get_avatar(message.author)
     nickname = message.author.display_name
     if message.channel.is_private:
-        os.system(
-            "notify-send -i {avatar} \"{author} in chat with {users}:\" {content}"
-            .format(
-                avatar=shlex.quote(avatar),
-                author=shlex.quote(nickname),
-                content=shlex.quote(message.clean_content),
-                users=shlex.quote(', '.join(a.display_name for a in message.channel.recipients))))
+        if PLATFORM == "Linux":
+            linux_notify("-i {avatar} \"{author} in chat with {users}:\" {content}"
+                         .format(
+                             avatar=shlex.quote(avatar),
```

Comment:
I would `shlex.quote` inside the notification functions instead.

Reply:
Hopefully fixed, I was worried about switching based on private message I guess.